libraries/fc/include/fc/log/logger.hpp

```diff
@@ -57,7 +57,9 @@ class Logger {
 
 #define STR(x) #x
 #define STRINGIFY(x) STR(x)
-#define LOCATION "[" __FILE__ ":" STRINGIFY(__LINE__) "] --"
+#define __FILENAME__ ( strrchr(__FILE__, '/') ? strrchr(__FILE__, '/') + 1 : __FILE__ )
+
+#define LOCATION "[" + std::string(__FILENAME__) + ":" + STRINGIFY(__LINE__) + "] --"
 
```

Comment:
I guess it is not ideal as it is creating new string, making copies etc during runtime... if some magic optimization does not take place...

Reply:
can you used STRINGIFY instead  std::string ? that's maybe better solution